packages/server/lambdas/alfresco/delete-folder.ts

```diff
@@ -8,20 +8,18 @@ import { DatabaseClient } from '../database/client';
 import { folderDeleteRequestBuilder } from './fileRequestBuilder';
 import fetch from 'node-fetch';
 import { RequestInit } from 'node-fetch';
-import { AlfrescoResponse } from './fileRequestBuilder/types';
 import { deleteComponent } from '../database/components/delete-node-component';
 
 const database = await DatabaseClient.build();
 
 let fileEndpointsCache: Array<CategoryDataBase> = [];
 
-const deleteFolder = async (options: RequestInit, nodeId: string): Promise<AlfrescoResponse | undefined> => {
+const deleteFolder = async (options: RequestInit, nodeId: string) => {
   const alfrescoCoreAPIUrl = `${getAlfrescoUrlBase()}/alfresco/versions/1`;
   const url = `${alfrescoCoreAPIUrl}/nodes/${nodeId}`;
   try {
     const res = await fetch(url, options);
-    const result = (await res.json()) as AlfrescoResponse;
-    return result;
+    return res;
   } catch (err) {
     console.error('error:' + err);
```

Comment:
Happened to notice while checking this other error stuff, that it's using console.error, not log.error.